src/createRollupConfig.ts

```diff
@@ -202,7 +202,6 @@ export async function createRollupConfig(
       sourceMaps(),
       shouldMinify &&
         terser({
-          sourcemap: true,
```

Comment:
As of [v6.0.0](https://github.com/TrySound/rollup-plugin-terser/releases/tag/v6.0.0), inferred automatically from Rollup's `output.sourcemap`